packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/RelationModal.tsx

```diff
@@ -352,8 +394,12 @@ const RelationModal = ({ children }: { children: React.ReactNode }) => {
         </Modal.Header>
         <Modal.Body>
           <FormContext
-            method="PUT"
-            initialValues={currentDocument.getInitialFormValues()}
+            method={isCreating ? 'POST' : 'PUT'}
+            initialValues={
+              isCreating
+                ? currentDocument.getInitialFormValues(isCreating)
+                : currentDocument.getInitialFormValues()
```

Comment:
Why not always call getInitialFormValues with isCreating? I assume it can handle isCreating being false

Reply:
I tried and I received an empty object